### PR TITLE
docs: explain how to set `directUrl`(#4778)

### PR DESCRIPTION
**Pull request by ankur-arch**
- explain why `directUrl` is needed
- demonstrate how to add `directUrl`

### CODE REVIEW

File: content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx [hunk 1]
```diff
@@ -119,11 +119,33 @@ You can generate and use Prisma Client for the Data Proxy in the following envir
...
+   ```
+
+   Then, add the direct database connection string in the `.env` file.
+
+   ```bash file=.env highlight=2;add
+   DATABASE_URL="prisma://****:****@****"
+   + DIRECT_URL="postgresql://****:****@****"
+   ```
+
+   However, in older versions, you need to overwrite the `DATABASE_URL` environment variable with the direct database connection string when running migrations. Refer to the [Prisma CLI commands with the Data Proxy](https://www.prisma.io/docs/data-platform/data-proxy/prisma-cli-with-data-proxy) documentation for more information.
```
Comment (janpio) [inline content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx:147]: This is now content under the headline `Generate Prisma Client for the Data Proxy`. Shouldn't this be under a different top level heading?

Comment (ankur-arch) [reply content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx:147]: The content was added to ensure users have a `directUrl` setup when generating the Prisma Client for Data Proxy. As step 1 replaces the database connection, step 2 indicates that migrations won't work without a direct URL and we show how to set that up. I agree it does not go well with the heading, I could move this into a new section below? 
@nikolasburk  @janpio - something like 'Using the direct database connection'

Comment (janpio) [reply content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx:147]: Yes, `directUrl` is only for Prisma CLI so maybe that can be used. The current headline uses "Prisma Client", so another one about using Prisma CLI in that situation would make sense.

Comment (nikolasburk) [reply content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx:147]: I agree that the headline isn't very accurate any more, nice catch Jan!

Maybe some of these work better now: 

- Configure project to use Data Proxy
- Configure Prisma project to use Data Proxy
- Configure local project to use Data Proxy
- Configure local Prisma project to use Data Proxy

Comment (ankur-arch) [reply content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx:147]: Changing the headline to  `Configure Prisma project to use Data Proxy`

**Pull request merged**